### ISSUE TITLE
extend: https://docs.julialang.org/en/v1.7-dev/manual/code-loading/#Definitions

### ISSUE THREAD

**Issue by GerhardTotschnig** [JuliaLang/julia#39136]
I would find it good to mention in the docs section:
https://docs.julialang.org/en/v1.7-dev/manual/code-loading/#Definitions
that including inside a function is not recommended and not what a beginner would expect:

https://discourse.julialang.org/t/included-code-behaves-different-than-code-in-place/52983/3
https://discourse.julialang.org/t/is-include-safe-to-use-inside-a-function/10867
https://discourse.julialang.org/t/how-to-include-into-local-scope/34634

**Comment by fredrikekre** [comment]
The first sentence of the docstring for `include` is

> Evaluate the contents of the input source file in the global scope of the containing module.

But perhaps that should be clarified/extended then?

**Comment by GerhardTotschnig** [comment]
Yes I read the docs a couple of times but for me it was not clear why
an include inside a function does not work. I am a Julia beginner.
Therefore I did post a Question on discourse and did get some answer:
https://discourse.julialang.org/t/included-code-behaves-different-than-code-in-place/52983/3 

I would find it helpful to mention more clearly that using include in a local scope construct
does not work as one my think.
